browser/js/items/item_page.html

```diff
@@ -8,7 +8,7 @@
 				<div class="col-lg-12">
 					<h3>{{product.productName}}</h3>
 					<p>Sold By: <a href="#">{{product.sellerName}}</a></p>
-					<button ng-repeat="tag in product.tags">{{tag}}</button>
+					<a class="tag_btn" ng-repeat="tag in product.tags">{{tag | uppercase}}</a>
```

Comment:
on line 4 of this file, put a class on the img that limits the height in case you have a narrow picture?